streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/application/impl/SparkApplicationManageServiceImpl.java

```diff
@@ -294,90 +283,83 @@ public boolean create(SparkApplication appParam) {
-        newApp.setExecutionMode(oldApp.getExecutionMode());
-        newApp.setSparkImage(oldApp.getSparkImage());
-        newApp.setK8sNamespace(oldApp.getK8sNamespace());
-        newApp.setK8sRestExposedType(oldApp.getK8sRestExposedType());
-        newApp.setK8sPodTemplate(oldApp.getK8sPodTemplate());
-        newApp.setK8sJmPodTemplate(oldApp.getK8sJmPodTemplate());
-        newApp.setK8sTmPodTemplate(oldApp.getK8sTmPodTemplate());
-        newApp.setK8sHadoopIntegration(oldApp.getK8sHadoopIntegration());
-        newApp.setDescription(oldApp.getDescription());
-        newApp.setAlertId(oldApp.getAlertId());
-        newApp.setCpFailureAction(oldApp.getCpFailureAction());
-        newApp.setCpFailureRateInterval(oldApp.getCpFailureRateInterval());
-        newApp.setCpMaxFailureInterval(oldApp.getCpMaxFailureInterval());
-        newApp.setMainClass(oldApp.getMainClass());
         newApp.setAppType(oldApp.getAppType());
+        newApp.setVersionId(oldApp.getVersionId());
+        newApp.setAppName(appParam.getAppName());
+        newApp.setExecutionMode(oldApp.getExecutionMode());
         newApp.setResourceFrom(oldApp.getResourceFrom());
         newApp.setProjectId(oldApp.getProjectId());
         newApp.setModule(oldApp.getModule());
-        newApp.setUserId(ServiceHelper.getUserId());
-        newApp.setState(FlinkAppStateEnum.ADDED.getValue());
-        newApp.setRelease(ReleaseStateEnum.NEED_RELEASE.get());
-        newApp.setOptionState(OptionStateEnum.NONE.getValue());
-        newApp.setHotParams(oldApp.getHotParams());
-
+        newApp.setMainClass(oldApp.getMainClass());
         newApp.setJar(oldApp.getJar());
         newApp.setJarCheckSum(oldApp.getJarCheckSum());
-        newApp.setTags(oldApp.getTags());
-        newApp.setTeamId(oldApp.getTeamId());
+        newApp.setAppProperties(oldApp.getAppProperties());
+        newApp.setAppArgs(appParam.getAppArgs() != null ? appParam.getAppArgs() : oldApp.getAppArgs());
+        newApp.setYarnQueue(oldApp.getYarnQueue());
+        newApp.resolveYarnQueue();
+        newApp.setK8sMasterUrl(oldApp.getK8sMasterUrl());
+        newApp.setK8sContainerImage(oldApp.getK8sContainerImage());
+        newApp.setK8sImagePullPolicy(oldApp.getK8sImagePullPolicy());
+        newApp.setK8sServiceAccount(oldApp.getK8sServiceAccount());
+        newApp.setK8sNamespace(oldApp.getK8sNamespace());
+
         newApp.setHadoopUser(oldApp.getHadoopUser());
+        newApp.setRestartSize(oldApp.getRestartSize());
+        newApp.setState(SparkAppStateEnum.ADDED.getValue());
+        newApp.setOptions(oldApp.getOptions());
+        newApp.setOptionState(OptionStateEnum.NONE.getValue());
+        newApp.setUserId(ServiceHelper.getUserId());
+        newApp.setDescription(oldApp.getDescription());
+        newApp.setRelease(ReleaseStateEnum.NEED_RELEASE.get());
+        newApp.setAlertId(oldApp.getAlertId());
+        newApp.setCreateTime(new Date());
+        newApp.setModifyTime(new Date());
```

Comment:
Whether the same Date object can be used

Reply:
done.